### DISCUSSION TITLE
Resolution change for  5 Inch LCD

### DISCUSSION THREAD

**Discussion by TH3050** [fayaaz/mixxx-pi-gen#84] [General]
Hello! I just wanted to ask if you could please help me. Trying to solve this for over 3 days now, to no avail...:))  I want to use this build with my Raspi and an 5 Inch LCD. I just can´t modifie the resolution. It loads ok till the GUI starts. With arandr I can´t do anything since it says that I don´t have Admin rights. I would greatly appreciate it.

**Comment by fayaaz** [comment]
Hello you can set the resolution in sway config or use wrandr - run it as root like:

`sudo wrandr`

To do it in sway try reading the docs with `man sway-output`

Apologies for the short and late reply, I am away but will check and write a guide in more detail when I'm back.

**Reply by RK11111111111** [reply]
For me I found that the resolution on the Pi was fine, but that some of the skins in Mixxx have a minimum resolution that was too large, artificially modifying the QT resolution wasn't successful.  Try one of the compact skins(if you can actually get to the menu option).  I rolled my own skin, 5in is pretty small what is the resolution, my screen was a 10.1in?  The pioneered skin was nice had a tabbed browser, but that may even be too large.  I have seen some ultra small skins on the web though.

**Comment by TH3050** [comment]
Thank you very much for the reply. I will try that what you recommend.

RK11111111111 ***@***.***> schrieb am Mi., 3. Apr. 2024,
19:07:
